syntax = "proto3";

option cc_generic_services = true; 

package mprpc;
message RpcHeader
{
    bytes service_name = 1;
    bytes method_name = 2;
    uint32 args_size = 3;
}